update.sh

```diff
@@ -9,25 +9,51 @@ if [ ${#versions[@]} -eq 0 ]; then
 fi
 versions=( "${versions[@]%/}" )
 
-packagesBase='http://apt.postgresql.org/pub/repos/apt/dists/jessie-pgdg'
-mainList="$(curl -fsSL "$packagesBase/main/binary-amd64/Packages.bz2" | bunzip2)"
+declare -A debianSuite=(
```

Comment:
I debated using a default, like [golang](https://github.com/docker-library/golang/blob/e72fd691a065bfec94de67bf826e4ebe511c4d24/update.sh#L4), but decided it just [complicates logic](https://github.com/docker-library/golang/blob/e72fd691a065bfec94de67bf826e4ebe511c4d24/update.sh#L114).